src/service/history-impl.js

```diff
@@ -589,9 +631,20 @@ export class HistoryBindingVirtual_ {
         this.onHistoryPopped_.bind(this));
   }
 
+  /**
+   * @param {string} target
+   * @override
+   */
+  replaceStateForTarget(target) {
+    this.win.location.replace(target);
+  }
+
   /** @override */
   cleanup_() {
     this.unlistenOnHistoryPopped_();
+    if (this.origReplaceState_) {
```

Comment:
Sorry this is not needed and is risidual from earlier experimentation. Will drop. We do not override either `pushState` or `replaceState` in virtual history.